docs/src/pages/components/data-grid/components/components.md

```diff
@@ -9,7 +9,7 @@ title: Data Grid - Components
 ## Overriding components
 
 As part of the customization API, the grid allows you to override internal components with the `components` prop.
-The prop accepts an object of type [`GridSlotsComponent`](/api/data-grid/#slots).
+The prop accepts an object of type [`GridSlotsComponent`](/api/data-grid/data-grid/#slots).
```

Comment:
**Off-topic**

We have talked about using these URLs in v5
```suggestion
The prop accepts an object of type [`GridSlotsComponent`](/api/x-data-grid/data-grid/#slots).
```

Using the name of the package then the name of the module exported: https://www.notion.so/mui-org/Sitemap-and-URL-structure-rearrange-ca604f628c17438db2d987961fcb8ffe

Reply:
Cool. I cannot access MUI notion. Is it possible to give me access?

Reply:
I've added another item to #2313 to keep track. We will need redirect rules to not send users to a 404. This is the proposed structure:

- [mui.com/components/x-data-grid](http://mui.com/components/x-data-grid) → X component main page.
    - [mui.com/api/x-data-grid/grid-toolbar](http://mui.com/api/x-data-grid/grid-toolbar) → Exposing smaller components with Data Grid, for example.
    - [mui.com/api/x-data-grid-pro/grid-toolbar](http://mui.com/api/x-data-grid/grid-toolbar) → Similarly, we can do it for those smaller components on the paid version.